pytorch/edgeml_pytorch/graph/rnn.py

```diff
@@ -967,78 +967,142 @@ class BaseRNN(nn.Module):
     [batchSize, timeSteps, inputDims]
     '''
 
-    def __init__(self, cell: RNNCell, batch_first=False):
+    def __init__(self, cell: RNNCell, batch_first=False, cell_reverse: RNNCell=None, bidirectional=False):
         super(BaseRNN, self).__init__()
-        self._RNNCell = cell
+        self.RNNCell = cell 
```

Comment:
Why did we flip self._RNNCell to self.RNNCell? Isn't it supposed to be private?

Reply:
I had similar doubt, but rnnpool flipped it so I too flipped for consistency. I can revert back.

Reply:
Ask @oindrilasaha what is the reason she made it public instead of private. We already pass in the cell, so we have access to it if needed.

Reply:
please make it as self._RNNCell everywhere, this change came since when i had pulled it was self.RNNCell.

Reply:
Done